gef-ui/server/controllers/application-submission/index.test.js

```diff
@@ -127,10 +138,10 @@ describe('controllers/application-submission', () => {
 
     it('renders confirmation if successfully submitted', async () => {
       await postApplicationSubmission(mockRequest, mockResponse);
-
       expect(mockResponse.render).toHaveBeenCalledWith('application-details-submitted.njk', expect.objectContaining({
         dealId: expect.any(String),
         submissionType: expect.any(String),
+        unissuedToIssued: expect.any(Boolean),
```

Comment:
Should this be in the test above as well? ^

This is the trouble with `expect.objectContaining({...` - if things are missing it'll still pass. Instead of checking the entire object - which would fail if anything is changed

Appreciate this is legacy code in your changes don't need to change the assertion

Reply:
"Should this be in the test above as well? ^" bit confused about this line.  Which test?

Reply:
Sorry in the same test there are two:
```js
expect(mockResponse.render).toHaveBeenCalledWith('....')
```

They use 2 different templates - not sure if they use the same variables? If they do, should be added to the test

Reply:
done